documentation/overview_and_index.md

```diff
@@ -0,0 +1,54 @@
+## Problem Statement
+[Human Rights First (HRF)](https://www.humanrightsfirst.org/)   is an independent advocacy and 
+action organization that challenges America to live up 
+to its ideals. HRF is seeking a data search and 
+visualization tool to provide the public with a 
+user-friendly view into their large data set of asylum 
+case decisions. Similar to [the TRAC database](https://trac.syr.edu/phptools/immigration/asylum/)—
+a popular existing tool, but with many limitations.
+
+This design document describes the HRF Asylum Report
+Generator backend, a new service that will allow users
+to view and filter decision data on multiple parameters,
+generate visualizations and download search results.
+
+## General Use Cases
+*For most up-to-date and detailed product requirements, please see Jira. This list is 
+intended to give an overall picture.*
+
+* A user can generate a table of results, filtered by multiple parameters
+* A user can generate a real-time graph to represent what trend looks
+like over time (Release 4)
+* A user can generate a heatmap of asylum case outcomes (Release 5)
+* A user can download the results of their search (Release 6)
+
+  
+## Project Scope
+
+### Current scope
+* Users can view all case data in a table
+* Users can filter the dataset by one or more parameters 
+
+
+### Future Scope
```

Comment:
I'm particularly fond of this distinction -- it makes the document a live document, but also gives it some longevity.